Multisig/UI/App/UIKit/AppSettingsViewController/AppSettingsViewController.swift

```diff
@@ -0,0 +1,226 @@
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let item = sections[indexPath.section].items[indexPath.row]
+        switch item {
+        case Section.General.importKey(let name):
+            return basicCell(name: name, indexPath: indexPath)
+        case Section.General.importedKey(let name, let signingKey):
+            return importedKeyCell(name: name, signingKey: signingKey, indexPath: indexPath)
+        case Section.General.terms(let name):
+            return basicCell(name: name, indexPath: indexPath)
+        case Section.General.privacyPolicy(let name):
+            return basicCell(name: name, indexPath: indexPath)
+        case Section.General.licenses(let name):
+            return basicCell(name: name, indexPath: indexPath)
+        case Section.General.getInTouch(let name):
+            return basicCell(name: name, indexPath: indexPath)
+        case Section.General.appVersion(let name, let version):
+            return infoCell(name: name, info: version, indexPath: indexPath)
+        case Section.General.network(let name, let network):
+            return infoCell(name: name, info: network, indexPath: indexPath)
+        case Section.Advanced.advanced(let name):
+            return basicCell(name: name, indexPath: indexPath)
+        default:
+            return UITableViewCell()
+        }
+    }
+
+    private func basicCell(name: String, indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueCell(BasicCell.self, for: indexPath)
+        cell.setTitle(name)
+        return cell
+    }
+
+    private func infoCell(name: String, info: String, indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueCell(InfoCell.self, for: indexPath)
+        cell.setTitle(name)
+        cell.setInfo(info)
+        cell.selectionStyle = .none
+        return cell
+    }
+
+    private func importedKeyCell(name: String, signingKey: String, indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueCell(ImportedKeyCell.self, for: indexPath)
+        cell.setName(name)
+        cell.setAddress(Address(exactly: signingKey))
+        cell.selectionStyle = .none
+        cell.onRemove = { [unowned self] in
+            self.removeImportedOwnerKey()
+        }
+        return cell
+    }
+
+    #warning("TODO: show snackbar message")
+    private func removeImportedOwnerKey() {
+        let alertController = UIAlertController(
+            title: nil,
+            message: "Removing the owner key only removes it from this app. It doesn't delete any Safes from this app or from blockchain. For Safes controlled by this owner key, you will no longer be able to sign transactions in this app",
+            preferredStyle: .actionSheet)
+        let remove = UIAlertAction(title: "Remove", style: .destructive) { [unowned self] _ in
+            do {
+                try App.shared.keychainService.removeData(
```

Comment:
I think from the design standpoint this code should be moved to the business logic layer - to something called `SigningController` or some other name

Reply:
there is a warning, I propose to refactor it with fixing the warning; I will add this note to the warning